test/e2e/predictor/test_raw_deployment.py

```diff
@@ -35,7 +35,6 @@ def test_raw_deployment_kserve():
     service_name = "raw-sklearn"
     annotations = dict()
     annotations['serving.kserve.io/deploymentMode'] = 'RawDeployment'
-    annotations['kubernetes.io/ingress.class'] = 'istio'
```

Comment:
Removing this annotation from test as it causes some errors:
```
annotations.kubernetes.io/ingress.class: Invalid value: \"istio\": can not be set when the class field is also set
```

Somehow the cluster or istio rejects when both `ingressClassName` and `annotations.kubernetes.io/ingress.class` are being set.